utils/superset/README.md

```diff
@@ -0,0 +1,24 @@
+# Script to rename the database name 
+
+The purpose of the script is to update the dashboard JSON template file with the database name of the user’s environment. 
+
+This script will generate the new upload JSON file with the database name of the user and will be ready to import to have the PGD dashboard.  
+
+Requires user to pass the database_name (which should be present in his/her environment) and optional input file and output file in JSON format as an argument in the script execution. The default extension is JSON for both input and output files. 
+
+# Requirements
+
+    Python 3.x 
+
+# Run Comamnd
+
+    python3 db_name_change.py -d <database_name> -i <input_file> -o <output_file>
```

Comment:
Should give an exmple using the utils/superset/import_template.json

Reply:
Done.